packages/addons/addon-depends/snapcast-depends/snapcast/package.mk

```diff
@@ -0,0 +1,42 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2018-present Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="snapcast"
+PKG_VERSION="0.13.0"
+PKG_SHA256="8409cdb20805d945fb243324a20a24b15c418466a2026bcc6a9211a6b32fdfae"
```

Comment:
checksum is incorrect - should be `331cfc62db8631af80f6c8798b9c359eb55e34d607af0881c3520b94f238f94e`

Reply:
I have this in a PR along with another couple of add-ons fixed, will post later